src/utils/utils.ts

```diff
@@ -338,25 +228,25 @@ export const tokenFromJSON = (token: any): OpenSeaFungibleToken => {
 
 // eslint-disable-next-line @typescript-eslint/no-explicit-any
 export const orderFromJSON = (order: any): Order => {
```

Comment:
Are we sure each of these are whole numbers to be able to use BIgNumber and not FixedNumber?

Reply:
unused fn removed